src/game_actor.cpp

```diff
@@ -45,6 +45,11 @@ Game_Actor::Game_Actor(int actor_id) :
 	Game_Battler(),
 	actor_id(actor_id) {
 	GetData().Setup(actor_id);
+	// - LSD readout is showing this happens.
+	GetData().class_id = -1;
+	GetData().battle_commands.clear();
+	for (int i = 0; i < 7; i++)
+		GetData().battle_commands.push_back(-1);
```

Comment:
48 to 52 can be removed as liblcf handles the correct initialisation now (or soon)